packages/ra-ui-materialui/src/layout/Notification.tsx

```diff
@@ -86,6 +86,7 @@ export const Notification = (
                     </StyledButton>
                 ) : null
             }
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
```

Comment:
I suggest to make the `NotificationProps` interface extends the `SnackbarProps` one from MUI. 
Then declare the `anchorOrigin` prop at line 25 and set its default to a this value.

And by the way, can you extract the `anchorOrigin` default value as a const outside the component?

Reply:
Yes of course. I think this is a good option!

I even commented on the issue

Reply:
I made the change https://github.com/marmelab/react-admin/pull/6878/commits/f1e0e79ce503ac0b89939bd40834122d3df06772, but it's also worth noting whether it makes sense now to add some prop in showNotification

Reply:
I will mark it as resolved because I changed this logic https://github.com/marmelab/react-admin/pull/6878#discussion_r755164822